common/src/lib.rs

```diff
@@ -1,21 +1,24 @@
 pub mod database;
+#[cfg(feature = "deployment")]
 pub mod deployment;
+#[cfg(feature = "log")]
 pub mod log;
 pub mod project;
 pub mod resource;
+#[cfg(feature = "secret")]
 pub mod secret;
+#[cfg(feature = "service")]
 pub mod service;
+#[cfg(feature = "version")]
 pub mod version;
-
+#[cfg(feature = "log")]
 pub use log::Item as LogItem;
```

Comment:
Maybe we should just have a single "models" feature for all the response models send between gateway/deployer and the client? We could even put them in a "models" or whatever namespace...